libs/sdk-ui-vis-commons/src/legend/StaticLegend.tsx

```diff
@@ -104,3 +135,6 @@ export class StaticLegend extends React.PureComponent<IStaticLegendProps> {
         );
     }
 }
+
+const shouldItemsFitOneColumn = (visibleItemsCount: number, columnNum: number, pagedSeriesLength: number) =>
```

Comment:
This is a bit confusing, so it returns false or "no-width"?

Reply:
fixed by 650ddd8